airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AdvancedAcceptanceTests.java

```diff
@@ -407,50 +373,6 @@ public void testCancelSyncWithInterruption() throws Exception {
 
   @RetryingTest(3)
   @Order(7)
-  @Timeout(value = 5,
-           unit = TimeUnit.MINUTES)
-  @EnabledIfEnvironmentVariable(named = "CONTAINER_ORCHESTRATOR",
-                                matches = "true")
-  public void testCuttingOffPodBeforeFilesTransfer() throws Exception {
```

Comment:
I removed this test entirely because it was almost identical to testDowntimeDuringSync